utils/load_mml.m

```diff
@@ -102,10 +111,12 @@ function load_mml(ringmode)
     function type = gettype(elem)
         if isfield(elem, 'Class')
             type = elem.Class;
-        elseif isfield(elem, 'FamName')
+            % We need to distinguish
+            % - BPMs, which have class 'Marker'
+            % - vertical and horizontal correctors
+            if (strcmp(type, 'Marker') || strcmp(type, 'Corrector')) && isfield(elem, 'FamName')
```

Comment:
It isn't very clear to me what this is diong

Reply:
Noted, but the whole script is a bit cryptic. I may try to improve it further.